src/Templates/src/templates/maui-mobile/Resources/Styles/Styles.xaml

```diff
@@ -28,6 +28,7 @@
         <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource Primary}, Dark={StaticResource White}}" />
         <Setter Property="FontFamily" Value="OpenSansRegular"/>
         <Setter Property="FontSize" Value="14"/>
+        <Setter Property="BorderWidth" Value="0"/>
```

Comment:
@davidortinau any complaints here? A possible different approach here would be to use `OnPlatform` to specify whatever the default is on `WinUI`

This will still keep them different but it's at least discoverable. 

Either approach probably accomplishes the same thing though, and this change is probably fine.